prov/verbs/src/verbs_ofi.h

```diff
@@ -933,6 +934,12 @@ do {								\
 	( wr->opcode == IBV_WR_SEND || wr->opcode == IBV_WR_SEND_WITH_IMM	\
 	|| wr->opcode == IBV_WR_RDMA_WRITE_WITH_IMM )
 
+struct vrb_recv_wr {
+	struct slist_entry	entry;
```

Comment:
nit: ibv_recv_wr has a next pointer which can be used to create a single-linked list.  Using that requires duplicating part of the slist implementation, so I'm not sure yet in the review if it's worth pursuing this change, as this is all non-critical path.

Reply:
I need some time to think about it. I'm not sure the change is worth it.